README.md

```diff
@@ -14,7 +15,8 @@ $ python app.py
 	- config.py # Configuration for development and testing environment
 	- models.py # SQLAlchemy models
 	- views.py # Flask routes
+    - search.py # helper functions to search
 - test
-	- __init__.py # Base testcase to prepare and destroy application and database
+	- \_\_init\_\_.py # Base testcase to prepare and destroy application and database
```

Comment:
If you want to quote code in markdown, you can put the thing between backticks (\`) just like `this`. See [GitHub's guide to markdown](https://guides.github.com/features/mastering-markdown/) for more.